examples/chicago_taxi/execute_on_flink.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -u
+
+echo Starting pipeline on Flink...
+
+if [ "${JOB_ENDPOINT:-unset}" == "unset" ]; then
+  JOB_ENDPOINT="localhost:8099"
+fi
+
```

Comment:
which steps here start the flink cluster?

Reply:
These scripts are simplified to use only a local flink cluster which has to be started separately. 
Also a separate beam job server has to be started separately based on https://beam.apache.org/roadmap/portability/#python-on-flink 

This script only submits the pipeline to beam job server which submits the job to the configured flink. 
Note: if no flink cluster is running locally then the beam job server will execute the flink job in process.

This script is not intended to be called by the users directly and is used by other scripts.